frontend/src/metabase/query_builder/actions/object-detail.js

```diff
@@ -108,17 +108,25 @@ export const loadObjectDetailFKReferences = createThunkAction(
       const queryResult = getFirstQueryResult(state);
 
       async function getFKCount(card, queryResult, fk) {
-        const fkQuery = Q_DEPRECATED.createQuery("query");
-
-        fkQuery.database = card.dataset_query.database;
-        fkQuery.query["source-table"] = fk.origin.table_id;
-        fkQuery.query.aggregation = ["count"];
-        fkQuery.query.filter = getFilterForFK(objectId, fk);
+        const metadata = getMetadata(getState());
```

Comment:
This part should probably have been migrated in #37639, but I missed it earlier.